src/runtime/offers-flow.js

```diff
@@ -120,18 +120,20 @@ export class OffersFlow {
     /** @private  @const {!Array<!string>} */
     this.skus_ = feArgsObj['skus'] || [ALL_SKUS];
 
-    /** @private @const {!Promise<!ActivityIframeView>} */
-    this.activityIframeViewPromise_ = this.getUrl_(
-      this.clientConfigManager_.getClientConfig()
-    ).then((url) => {
-      return new ActivityIframeView(
-        this.win_,
-        this.activityPorts_,
-        feUrl(url),
-        feArgsObj,
-        /* shouldFadeBody */ true
-      );
-    });
+    /** @private @const {!Promise<!ActivityIframeView|null>} */
```

Comment:
is "!ActivityIframeView|null" different from "?ActivityIframeView"? I thought the "?" indicates "|null".

Reply:
I was intending for explicit null (v.s. undefined like typescript) but, it doesn't seem to make a difference here. The docs are more explicit for behavior for params but I'll move to the simpler syntax in any case.